source/blender/editors/space_logic/logic_window.c

```diff
@@ -985,41 +985,34 @@ static void draw_sensor_delay(uiLayout *layout, PointerRNA *ptr)
 
```

Comment:
Basically, with the new system used by SDL2 every joystick/gamepad/etc is configured as a xbox360 controller. Then there is no necessity to expose old elements, i.e.:

- There is no hats, now the digital pads are treated as normal buttons.
- We only expect 15 buttons max
- etc

Then I modified the UI to show the buttons and axis as in the Xbox360 controller to facilitate the configuration.

The issue with that new SDL2 system is that if your joystick is not preconfigured you have to use Steam Big Mode, controllermapping in SDL2 test sources or antimicro application to generate a new valid mapping. But it seems that it is being very well accepted by the community.